packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/lib_auth/blocs/user_account_bloc.dart

```diff
@@ -63,7 +64,16 @@ class UserAccountBloc extends $UserAccountBloc {
   ConnectableStream<bool> _mapToLoggedInState() => Rx.merge([
         _coordinatorBloc.states.isAuthenticated,
         _authService.isAuthenticated().asStream(),
-      ]).publishReplay(maxSize: 1);
+      ]){{#enable_pin_code}}
+      .doOnData((isUserLoggedIn) {
+        if(isUserLoggedIn){
+          _coordinatorBloc.events.checkIsPinCreated();
```

Comment:
I think this is not the intended use for the `CoordinatorBloc`. It should forward events that occur inside the application, like `isAuthenticated`, etc. After that whoever is interested in something can listen to the coordinator bloc states and react accordingly. Having calls to `checkIsPinCreated()` or `deleteStoredPin()` introduces PIN code related logic inside the both the CoordinatorBloc and UserAccountBloc, which I think can be avoided.

Reply:
The named of the events and states in coordinator bloc related to user log in and log out were updated